File: Java/Userinput/src/uinput.java
import java.util.Scanner;

public class uinput {

	public static void main(String[] args) {
		
		Scanner scanner = new Scanner(System.in);
		
		System.out.println("What is your name? ");
		String name = scanner.nextLine();
		
		System.out.println("\nHello "+name);
		
		System.out.println("\nHow old are you? ");
		int age = scanner.nextInt();
		scanner.nextLine();
		
		System.out.println("\nHello "+name+". You are " +age+" years old.\n");
		
		System.out.println(name+", What is your favorite food?");
		String favfood = scanner.nextLine();
		
		System.out.println("\n"+favfood+"? I love that food!");
		
	}

}

File: C#/userinput/uinput.cs
namespace user_input;

class UserInput
{
    static void Main(string[] args)
    {   
        Console.WriteLine("What is your name?");
        String name = Console.ReadLine();
        Console.WriteLine("Hello " +name);
        
        Console.WriteLine("\nHow old are you?");
        int age = Convert.ToInt32(Console.ReadLine()); // Converting to an integer first is reccomended
        Console.WriteLine("Hello " +name);
        Console.WriteLine("You are " +age +" years old");
    }
}